.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Docker Image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think you can use actions/checkout@v3 here

Reply:
Thanks for the heads up!